src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -0,0 +1,205 @@
+import React, { useState, useRef } from 'react';
+import styled from 'styled-components';
+import { Button } from '../Button/Button';
+import { mockedAuthorsList, mockedCoursesList } from '../../utils/constants';
```

Comment:
Are mockedAuthorsList and mockedCoursesList imports used in this file?
Consider adding `airbnb` eslint configs to your projects as they are able to detect unused imports